operator/src/main/resources/scripts/startServer.sh

```diff
@@ -102,6 +102,11 @@ function waitUntilShutdown() {
     trace "Showing the server out file from ${SERVER_OUT_FILE}"
     ${SCRIPTPATH}/tailLog.sh ${SERVER_OUT_FILE} &
   fi
+  FAIL_BOOT_ON_SITUATIONAL_CONFIG_ERROR=${FAIL_BOOT_ON_SITUATIONAL_CONFIG_ERROR:-true} 
+  SERVER_OUT_MONITOR_INTERVAL=${SERVER_OUT_MONITOR_INTERVAL:-30}
```

Comment:
Since the monitor script only runs until the server reports that it started successfully - (I don't recall the original change did that - maybe I missed that) - it seems like it'd be OK to run the monitor more frequently than every 30 seconds.

Reply:
It was added in the latest change. any suggestion on interval? 10 seconds?

Reply:
Maybe even back to 3 seconds?  Boots can occur pretty quick these days.

Reply:
Updated interval back to 3 seconds.